packages/schemas/tables/settings.sql

```diff
@@ -0,0 +1,13 @@
+create type tenant_region_type as enum ('china_mainland', 'china_hongkong', 'china_taiwan', 'united_states', 'japan');
+
+create table settings (
+    tenant_id varchar(24) not null,
+    tenant_name text not null,
+    tenant_region tenant_region_type not null default('china_mainland'),
+    skip_get_started boolean not null default('true'),
+    custom_domain text not null,
```

Comment:
this can be null actually. say we're testing Logto locally, there's no `custom_domain`.